integration.sh

```diff
@@ -50,11 +50,11 @@ function run_tests() {
 	elif [[ $version == 2.2.* || $version == 3.0.* ]]; then
 		proto=4
 		ccm updateconf 'enable_user_defined_functions: true'
-		export JVM_EXTRA_OPTS=" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"
+		export JVM_EXTRA_OPTS=" -Dcassandra.test.fail_writes_ks=test -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"
```

Comment:
actually, not sure if you need this here as the test will skip if the version is < 3.11.x

Reply:
I thought I had removed that `Skip`. I realized Cassandra prior to 3.11 actually had support for that when the tests pass when the `Before` was inverted.